### ISSUE TITLE
100% Dimmer Level 

### ISSUE THREAD

**Issue by simonrb2000** [PatchworkBoy/homebridge-edomoticz#168]
Hi, Since the latest Domoticz Stable, HomeKit shows all dimmers as 100% when they are on, even if they are not set to 100%. 

I think this is a domoticz issue and only on some type of dimmers. I'll have to check what values show up for each device. Just wondered if anyone else has this issue?

Many Thanks

**Comment by PatchworkBoy** [comment]
Presuming resolved - cannot duplicate issue, no further posters mentioning issue.

**Comment by simonrb2000** [comment]
This still happens. Basically when they are turned on Domoticz displays them as I’m rather than a value. So I’m HomeKit they show as 100%. Is there a way for the dimmers to still get value rather than assume 100% as Domoticz says on?

**Comment by PatchworkBoy** [comment]
Not quite understanding what you’re asking... that the Domoticz UI says 100% instead of "On"?

If I’m reading this right, this is if you have a dimming value set in Homebridge, turn the light off, turn the light on again, Domoticz will merely state "On"... correct? If so that’s because it’s the last command received by Domoticz - the dim hasn’t changed, just an On or Off sent. To have Domoticz ALWAYS show dim percentage instead of "On" is a Domoticz issue, not a homebridge-edomoticz issue. In Homebridge, I’m always seeing the percentage.

**Comment by simonrb2000** [comment]
That’s correct. I don’t mind domoticz showing as on when a dimmer is turned on. The issue is that in HomeKit the dimmer shows as 100% when domoticz says on rather than the correct percent.. is it they way that edomoticz issues the on command? It didn’t used to behave this way. If I turned a light on through HomeKit it remembered the previous value. So if it was 80% it came back on and displayed 80%. The change in behaviour now when turning on a dimmer from HomeKit it says on in domoticz and 100% in HomeKit however the light is correctly at 80% and not 100%.


Cheers



On 4 Jan 2020, at 16:30, Marci <notifications@github.com> wrote:

﻿

Not quite understanding what you’re asking... that the Domoticz UI says 100% instead of "On"?

If I’m reading this right, this is if you have a dimming value set in Homebridge, turn the light off, turn the light on again, Domoticz will merely state "On"... correct? If so that’s because it’s the last command received by Domoticz - the dim hasn’t changed, just an On or Off sent. To have Domoticz ALWAYS show dim percentage instead of "On" is a Domoticz issue, not a homebridge-edomoticz issue. In Homebridge, I’m always seeing the percentage.

**Comment by PatchworkBoy** [comment]
I’m still not seeing this behaviour... all behaves and is reflected perfectly here. *baffled*

**Comment by Ant1q** [comment]
Hello! Have problem with dimmer - it adds 1% for level in HomeApp.
For example: i set lamp to 50%, in domoticz i get 51%, and when i want set 100%, domoticz gets 101%, and there is problem - dimmers do not work correctly with this value.
